schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml

```diff
@@ -206,6 +206,11 @@
       <summary>Provide workspace isolation</summary>
       <description>Dash shows only windows from the currentworkspace</description>
     </key>
+    <key type="b" name="show-urgent-windows">
```

Comment:
Maybe should be called something `always-show-urgent-windows`? Or `workspace-agnostic-urgent-windows`?